src/QMCWaveFunctions/Jastrow/RadialJastrowBuilder.cpp

```diff
@@ -348,8 +348,8 @@ std::unique_ptr<WaveFunctionComponent> RadialJastrowBuilder::createJ1(xmlNodePtr
   using TH     = JastrowTypeHelper<RadFuncType, Implementation>;
   using J1Type = typename std::conditional<SPIN, typename TH::J1SpinType, typename TH::J1Type>::type;
 
-  XMLAttrString input_name(cur, "name");
-  std::string jname = input_name.empty() ? Jastfunction : input_name;
+  std::string input_name(getXMLAttributeValue(cur, "name"));
+  std::string jname = !input_name.empty() ? input_name : Jastfunction;
```

Comment:
Prefer changing `!empty()` back to `empty()`.

Reply:
These are returned to there original configuration.